### ISSUE TITLE
Missing firmware (bios) 1.65 update for lenovo T470

### ISSUE THREAD

**Issue by goebbe** [fwupd/firmware-lenovo#92]
Firmware (bios) currently available in LVFS:   1.64
Device homepage on LVFS: 
https://fwupd.org/lvfs/search?value=T470

The newest firmware (bios) currently available from vendor:   **1.65**
Since: 11 Nov 2020

Recommended Bios update available as a download from vendor support page: 
https://pcsupport.lenovo.com/ar/de/products/laptops-and-netbooks/thinkpad-t-series-laptops/thinkpad-t470/downloads/ds120429

**Comment by goebbe** [comment]
Now, I just **manually** updated the BIOS of T470, using the .cab-files directly from the Lenovo support page.
Using fwupdmgr.
`~$ sudo fwupdmgr install N1QET90W.cab`

After rebooting, it seems that the Bios update (available from Lenovo) seems to work fine: 
```
~$ sudo dmidecode -t bios
...    
BIOS Information
	Vendor: LENOVO
	Version: N1QET90W (1.65 )
```

The working BIOS update is missing from LVFS/ has not been uploaded. At least the follwoing command did **not find** the firmware update: 
```
sudo fwupdmgr refresh
sudo fwupdmgr get-updates
```

**Comment by goebbe** [comment]
This seems to be fixed, since 2021-02-22 05:01:07.815396.
[LVFS](https://fwupd.org/lvfs/devices/com.lenovo.ThinkPadN1QET.firmware)
Thank you!

**Comment by goebbe** [comment]
The firmware update is now uploaded to LVFS.  
closing the issue.
